src/Pages/yeheum/Login/Login.js

```diff
@@ -30,28 +27,37 @@ class Login extends React.Component {
     this.props.history.push("/main-yeheum");
   };
 
-  postToLocal = () => {
-    fetch("", {
-      method: "GET",
-      headers: {
-        Authorization: token,
-      },
+  handleClickPost = () => {
+    const { email, pw } = this.state;
+    fetch("http://10.58.4.237:8000/user/signin", {
+      method: "POST",
       body: JSON.stringify({
-        id: "kim",
-        password: "1234",
+        email: email,
```

Comment:
객체 단축 속성명을 사용하면 객체의 key와 value의 변수명이 같을 때 축약해줄 수 있습니다
```suggestion
        email,
```